non verbose defaults (#1566)
ROCmSoftwarePlatform/rocBLAS: docs/how-to/logging-in-rocblas.rst

program exits abnormally, it is possible that profile logging will
not sent to the output before the program exits.

GEMM backend logging
====================

> To produce additional logging to try and understand non success return codes,
> you may enable verbose error messages for the two backend systems used to perform GEMMs.

Reviewer: ```suggestion
To generate additional logging to analyze non-success return codes,
you can enable verbose error messages for the two backend systems used to perform GEMMs.
```

.. code-block:: shell

   export ROCBLAS_VERBOSE_TENSILE_ERROR=1
   export ROCBLAS_VERBOSE_HIPBLASLT_ERROR=1

> These may be used in conjunction with ``ROCBLAS_LAYER=8`` to improve understanding of an error,

Reviewer: ```suggestion
These can be used in conjunction with ``ROCBLAS_LAYER=8`` for a better understanding of an error,
```

> or even with success status to understand why one backend was not used.

Reviewer: ```suggestion
or even with a success status to understand why a backend was not used.
```
